### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by gitmyth**
testing hotfix merge

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1 +1,7 @@
-This is a test readme
\ No newline at end of file
+This is a test readme
+Hello
+World
+Hello
+there
+Testing
+hotfix
```
Comment (trinipi-dev) [inline readme.md:7]: Fix Title case

**Review by trinipi-dev — CHANGES_REQUESTED** [review]
Request to review comments and provide appropriate changes

File: readme.md (hunk 1 above)
Comment (gitmyth) [reply readme.md:7]: Will be reviewed and corrected in an upcoming build

**Review by trinipi-dev — APPROVED** [review]
Title Case verified, proceed to merge with main branch

**Pull request merged**